packages/logger/test/ogma.spec.ts

```diff
@@ -143,7 +143,7 @@ OgmaSuite(
     const loggedVal = getFirstCallString(writeSpy);
     match(loggedVal, /\[Circular\]/);
     match(loggedVal, /\[Function:/);
-    match(loggedVal, /\[Symbol\(hello\)\]/);
+    match(loggedVal, 'hello');
```

Comment:
What caused this to no longer print out `[Symbol(hello)]`?

Reply:
Doing some debugging, looks like something isn't quite what we're expecting right here:
![image](https://user-images.githubusercontent.com/28268680/213585069-4e4298bc-546b-4e89-9a49-1b3df1dcdab2.png)

Unless this is a part of the breaking changes. `symbol.toString()` already makes the value be `Symbol(value)` so we may want to revisit what the circular replacer is returning for symbols

Reply:
We can change to `symbol.description` instead, so it will not duplicate the value `Symbol`.

Reply:
With how I originally had it written, logging out, `Symbol(value)`, I was mimicking the behavior of `console.log` (while adding the brackets around it). Now, we'd be making our own format, but being consistent within the logger itself as `BigInt` and `Function` already use this `key: value` syntax :thinking:

Yeah, I think let's change it to `symbol.description` and note it in the breaking changes

Reply:
Let's also update the test to show the full syntax being checked for, for the sake of verification and security in future updates